onboard/src/pages/subsections/backgroundInformation/Personalnformation.jsx

```diff
@@ -1,63 +1,362 @@
-import React from "react"
-import TextField from "@mui/material/TextField";
-
-function PersonalInformation () {
-    return (
-        <>
-            <TextField
-              id="outlined-basic"
-              label="First Name"
-              variant="outlined"
-            />
-
-            <TextField
-              id="outlined-basic"
-              label="Last Name"
-              variant="outlined"
-            />
-
-            <TextField
-              id="outlined-basic"
-              label="Middle Initial"
-              variant="outlined"
-            />
-
-            <TextField
-              type="number"
-              id="outlined-basic"
-              label="SSN"
-              variant="outlined"
-            />
-            <TextField
-              type="date"
-              id="outlined-basic"
-              label="Birthdate"
-              variant="outlined"
-            />
-            <TextField
-              id="outlined-basic"
-              label="Current Address"
-              variant="outlined"
-            />
-            <TextField
-              id="outlined-basic"
-              label="Permanent Address"
-              variant="outlined"
-            />
-            <TextField
-              type="number"
-              id="outlined-basic"
-              label="Primary Phone Number"
-              variant="outlined"
-            />
-            <TextField
-              type="number"
-              id="outlined-basic"
-              label="Secondary Phone Number"
-              variant="outlined"
-            />
-          </>
-    );
+import React, { useState } from "react";
+import Grid from "@mui/material/Grid";
+import Box from "@mui/material/Box";
+import { FormControl } from "@mui/material";
+import InputField from "../../../components/inputs/InputField";
+import {
+  validAddress,
+  validPhoneNum,
+  validSSN,
+} from "../../../components/Regex";
+
+function PersonalInformation() {
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [middleInital, setMiddleInital] = useState("");
+  const [phoneNum, setPhoneNum] = useState("");
+  const [secondaryPhoneNum, setSecondaryPhoneNum] = useState("");
+  const [ssn, setSSN] = useState("");
+  const [dateOfBirth, setDateOfBirth] = useState(new Date());
+  const [currentAddress, setCurrentAddress] = useState("");
+  const [permanentAddress, setPermanentAddress] = useState("");
+
+  //const [dateOfBirthHelper, setDateOfBirthHelper] = useState("");
```

Comment:
Same as above, we ought to apply some type of regex to the date and have error handling.